SPECIFICATION.md

```diff
@@ -158,6 +159,73 @@ be on their own line.
 To prevent reference loops and to facilitate single-pass parsers, only backward
 references are allowed. All references glyphs must have been previously defined.
 
+#### Whitespaces
+
+Some glyphs need additional whitespace at the right of the glyph, or even
+consist of only whitespace (no polylines). The width of this whitespace can be
+specified for every glyph with the "S" character (for "space"), followed by a
+width in the same unit as polylines.
+
+The most important space glyph (which every font file MUST contain) is " "
+(U+0020 SPACE). A typical font may use a width of 3.6 (40% of font height):
+
+    [0020] SPACE
+    S3.6
+
+As global letter spacing is applied to every glyph (including whitespace glyphs),
+the total whitespace is the addition of both values (e.g. 3.6 + 1.8 = 5.4).
+
+Whitespace definitions are also inherited when referencing other glyphs (with
+"@", see above). If a glyph has multiple whitespace definitions (either direct
+or via references), only the last definition will be used by the parser.
+
+
+## Letter Spacing / Kerning
+
+Generally the space between letters depends on the specific use-case and thus
+it's up to the application's font layout engine to determine the exact space. An
+application may even implement proper [kerning](https://en.wikipedia.org/wiki/Kerning)
+if desired.
+
+But because this is too complex for many use-cases, FontoBene allows to specify
+some spacing information directly in font files. This information can be used by
+simple layout engines to get nice looking texts with very low effort. There are
+three different options which allow to control letter spacing:
+
+1. **Global Letter Spacing:**
+
+   The parameter `letter_spacing` in the `[font]` section of the header defines
+   the letter spacing which should be put after every letter.
+
+2. **Left spacing of glyphs:**
```

Comment:
Do we want to support right-to-left fonts in the future? If yes, "right" and "left" should maybe be replaced with "end" and "start". (That's the terminology that Android uses too in its UI files.)

Granted, we also have other problems if we want to support RTL, it's not the only issue :)

Reply:
I opened #34 for that.